.github/CODEOWNERS

```diff
@@ -1,29 +1,29 @@
 # Default code owners for entire repository
```

Comment:
@rbarkerSL, I've committed your suggestions but I also noticed that the @hashgraph/platform-base and @hashgraph/hedera-services teams don't seem to be valid here according to GitHub. Could you please take a look?

Reply:
@PavelSBorisov I have resolved this issue by adding the two groups to the repository access.